modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -121,110 +105,418 @@ Note that this applies to all variations of CSV files (see examples below for ot
 .Query
 [source, cypher]
 ----
-LOAD CSV FROM 'https://data.neo4j.com/bands/artists.csv' AS line
-CREATE (:Artist {name: line[1], year: toInteger(line[2])})
+LOAD CSV FROM 'https://data.neo4j.com/bands/artists.csv' AS row
+MERGE (a:Artist {name: row[1], year: toInteger(row[2])})
+RETURN a
 ----
 
 .Result
-[role="queryresult"]
+[source, role="queryresult"]
+----
++-----------------------------------------------+
+| a                                             |
++-----------------------------------------------+
+| (:Artist {name: "ABBA", year: 1992})          |
+| (:Artist {name: "Roxette", year: 1986})       |
+| (:Artist {name: "Europe", year: 1979})        |
+| (:Artist {name: "The Cardigans", year: 1992}) |
++-----------------------------------------------+
+Added 4 nodes, Set 8 properties, Added 4 labels
+----
+====
+
+
+=== Import compressed CSV files
+
+`LOAD CSV` can read local CSV files compressed with ZIP or gzip.
+ZIP archives can have arbitrary directory structures but may only contain a single CSV file.
+
+.Import a CSV file from within a ZIP file
+[source, cypher]
 ----
-+-------------------+
-| No data returned. |
-+-------------------+
-Nodes created: 4
-Properties set: 8
-Labels added: 4
+LOAD CSV FROM 'file:///artists.zip' AS row
+MERGE (:Artist {name: row[1], year: toInteger(row[2])})
 ----
 
 
-[[load-csv-import-data-from-a-csv-file-containing-headers]]
-== Import data from a CSV file containing headers
-When your CSV file has headers, you can view each row in the file as a map instead of as a `LIST<STRING>`.
+=== Import data from relational databases
 
-.artists-with-headers.csv
-[source, csv, role="noheaders", filename="artists-with-headers.csv"]
+If the source data comes from a relational model, it's worth evaluating how to gain the most from moving to a graph data model.
+Before running the import, think about how the data can be modeled as a graph, and adapt its structure accordingly when running the import (see link:https://neo4j.com/docs/getting-started/data-modeling/guide-data-modeling/[Graph data modeling]).
+
+Data from relational databases may consist of one or multiple CSV files, depending on the source database structure.
+A performant approach is to run multiple passes of `LOAD CSV` to import nodes separately from relationships.
+
+
+.Import from a single CSV file
+====
+The source file link:https://data.neo4j.com/importing-cypher/books.csv[_books.csv_] contains information about both authors and books.
+From a graph perspective, these are nodes with different labels, so it takes different queries to load them.
+
+The example executes multiple passes of `LOAD CSV` on that one file, and each pass focuses on the creation of _one_ entity type.
+This is the most performant practice, see <<_separate_creation_of_nodes_and_relationships>>.
```

Comment:
Looks the <<>> part isn't rendering correctly

Reply:
good catch :) it refers to a section we commented out. gonna update